ddw_analyst_ui/docker_settings.py

```diff
@@ -6,8 +6,9 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'postgres',
-        'USER': 'postgres',
+        'NAME': 'analyst_ui',
+        'USER': 'analyst_ui_user',
+        'PASSWORD': 'analyst_ui_pass',
```

Comment:
I think we should really abstract this... not secure to include it in the repo. I understand if this is just to ease development. I don't mind if we have to manually update this file before deployment.

Reply:
I've included a `local_settings.py.example` file in `ddw_analyst_ui.` If we rename this to `local_settings.py` it can override any settings module as long as it's imported in the right place, and it's gitignored.